packages/components/src/components/chat/ContextContainer.vue

```diff
@@ -217,7 +221,12 @@ export default Vue.extend({
       this.closeMenu();
     },
     onClickHeader() {
-      if (!this.isCollapsable) return;
+      if (!this.isCollapsable) {
+        if (this.isFile) {
+          this.$root.$emit('open-location', this.location, this.directory);
+        }
+        return;
+      }
```

Comment:
At the moment there's no indication that these are clickable. Consider adding the `cursor: pointer` style to the element.

Reply:
Will do.